WordPress/src/main/res/values/strings.xml

```diff
@@ -2385,6 +2388,7 @@
     <string name="me_profile_photo">Profile Photo</string>
     <string name="me_btn_app_settings">App Settings</string>
     <string name="me_btn_support">Help &amp; Support</string>
```

Comment:
and this me_btn_support

Reply:
Removed: e1f1d73a54abf855e7f4de45414f98916a64d4b5